redox-w-keyboard-basic/program_left.sh

```diff
@@ -1,12 +1,12 @@
 #!/bin/bash
 echo '=============================== MAKING ================================'
 cd custom/armgcc
-make
+make keyboard_side=left
```

Comment:
could you not just make the keyboard side a required parameter then you wouldn't need 2 programming scripts ?

Reply:
Sure just wanted not rework everything... But in that case I would propose to remove the scripts completely and add programming targets to the make files. What do you think?

Reply:
I think the two programming script and the main makefile are good right now. I would ask you to update the readme with the instruction to use the scripts you added instead.